scripts/buildDocs.js

```diff
@@ -32,30 +48,22 @@ components.forEach(component => {
     console.error(`${componentName} has invalid category "${component.category}"`);
   }
 
-  let content = '';
+  /* First Tab */
 
-  /* Markdown Front Matter */
-  content += `---\n`;
-  if (isParentComponent) {
-    content += `sidebar_position: 1\n`;
-  }
-  content += `id: ${component.name}\n`;
-  content += `title: ${isIncubatorComponent ? 'Incubator.' : ''}${component.name}\n`;
-  content += `sidebar_label: ${componentName}\n`;
-  content += `---\n`;
+  let firstTabContent = '';
```

Comment:
Since this script became big and we have more plans for it in the future, maybe it's time to organize it a little. 

Let's create a dedicate method for creating the first tab, later on we'll add more methods for the other tabs, but overall it will make the script more readable

Reply:
Done